script.js

```diff
@@ -1,70 +1,96 @@
-}
-// eslint-disable-next-line no-unused-vars
-function Clear() {
-  localStorage.clear();
-  Books = [];
-  document.getElementById('list_container').innerHTML = '';
-}
-function CreateHtml(IdBook) {
-  const newdiv = document.createElement('div');
-  newdiv.setAttribute('id', IdBook);
-  const title = document.createElement('p');
-  const author = document.createElement('p');
-  const RemoveButt = document.createElement('button');
-  RemoveButt.addEventListener('click', () => { NewbookRemove(RemoveButt); });
-  RemoveButt.textContent = 'remove';
-  RemoveButt.setAttribute('id', IdBook);
-  return [newdiv, title, author, RemoveButt];
-}
-function SaveLocally(BookArray) {
-  const BookList = JSON.stringify(BookArray);
-  localStorage.setItem('library', BookList);
-}
-function RetrieveOld() {
-  Books = JSON.parse(localStorage.getItem('library')) || [];
-  SaveLocally(Books);
-}
+class BookList {
+  constructor(Books) {
+    this.BookListCollection = Books;
+  }
+
+  clear() {
+    localStorage.clear('library');
+    document.getElementById('list_container').innerHTML = '';
+    this.BookListCollection = [];
+  }
 
-function ShowBook() {
-  RetrieveOld();
-  const BooksCurrent = JSON.parse(localStorage.getItem('library'));
   /* eslint-disable */
-  BooksCurrent.forEach(element => {
-    const x = CreateHtml(element.id);
-    x[1].innerHTML = `Title:   ${element.title}`
-    x[2].innerHTML = `Author:   ${element.author}`;
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[1]);
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[2]);
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[3]);
-  });
+  AddBook(Book) {
+    this.BookListCollection.push(Book);
+    const TitleP = document.createElement('p');
+    const AuthorP = document.createElement('p');
+    const RButton = document.createElement('button');
+    RButton.addEventListener('click', () => { this.RemoveBook(Book.id); });
+    RButton.innerHTML = 'Remove';
+    RButton.setAttribute('id', Book.id);
+    let BookContainer = document.createElement('div');
+    BookContainer.setAttribute('id', Book.id);
+    TitleP.innerHTML = `Title:   ${Book.title}`;
+    AuthorP.innerHTML = `Author:   ${Book.author}`;
+    document.getElementById('list_container').appendChild(BookContainer).appendChild(TitleP);
+    document.getElementById('list_container').appendChild(BookContainer).appendChild(AuthorP);
+    document.getElementById('list_container').appendChild(BookContainer).appendChild(RButton);
+  }
   /* eslint-enable */
+
+  RemoveBook(BookId) {
+    const BookContainer = document.getElementById(BookId);
+    BookContainer.parentNode.removeChild(BookContainer);
+    const BooksNew = [];
+    /* eslint-disable */
+    for (const i in this.BookListCollection) {
+      if (this.BookListCollection[i].id !== BookId) {
+        BooksNew.push(this.BookListCollection[i]);
+      }
+      /* eslint-enable */
```

Comment:
- [ ] Avoid disabling the entire eslint ruleset. Please fix the issue on the code instead as that improves the code quality.